common-rs/src/stake_accounts.rs

```diff
@@ -213,7 +214,7 @@ fn map_stake_accounts_to_settlement(
             }
         }
     }
-    // sorting stake accounts that the delegated ones are before the non-delegated ones
+    // sorting stake accounts for the delegated ones being before the non-delegated ones
```

Comment:
This is a bit of left-over that I decided to leave here when the functionality will be enhanced.

The idea behind is that when the settlement could be funded from two sources - from validator bond and from marinade then the claiming should prioritize the validator bond source. Marinade funds Initialized non delegated stake accounts so that should be moved at the end of the list.

Anyway, not needed not, it should be rather removed as makes rather a code buzz here.